docs/README.md

```diff
@@ -229,6 +229,10 @@ Here's an example configuration file for advanced settings and all of it's possi
         min: 1
         max: 1
         message: 'Custom message...'
+      
+      dependent:
+        files: [] // list of files that all must be modified if one is modified
```

Comment:
Would be good to give an example of array i.e. `['package.json', 'yarn.lock'] 

Also comment in YML is `#` :-)